jwst/regtest/conftest.py

```diff
@@ -399,13 +400,20 @@ def fitsdiff_default_kwargs():
     dict
         Keyword arguments to pass into FITSDiff.
     """
+    if not RELAX_TOL:
+        rtol = 1e-5
+        atol = 1e-7
+    else:
+        rtol = 1e-5
+        atol = 0.01
```

Comment:
Looks like this got triggered correctly by env var, but `{..., 'rtol': 1e-05, 'atol': 0.01}` is still insufficient. I will have to tweak the numbers again when I have a chance.